locales/DE_de.json

```
@@ -0,0 +1,139 @@
+			},
+			{
+				"question": "Werden meine vorhandenen NNIDs mit Pretendo funktionieren?",
+				"answer": "Leider nicht. Vorhandene NNIDs werden nicht mit Pretendo funktionieren, da nur Nintendo deine Nutzerdaten hat. Auch wenn eine NNID-zu-PNID Migration theoretisch möglich ist, wäre es riskant und würde Sensitive Nutzerdaten erfordern, die wir nicht Speichern möchten."
+			},
+			{
+				"question": "Wie benutze ich Pretendo?",
+				"answer": "Pretendo ist aktuell nicht bereit für die Öffentliche Nutzung. Allerdings, sobald Pretendo bereit ist, kannst du es einfach durch unseren homebrew patcher auf deiner Konsole nutzen."
+			},
+			{
+				"question": "Weißt du, wann <Funktion/Dienst> bereit ist?",
+				"answer": "Nein. Viele von Pretendo's Funktionen und Diensten werden unabhängig voneinander entwickelt (zum Beispiel, ein Entwickler arbeitet am Miiverse während ein anderer an Accounts und der Freundesliste arbeitet) und deshalb können wir keine Zeit nennen, wie lange es noch dauern wird."
+			},
+			{
+				"question": "Funktioniert Pretendo mit Cemu oder Emulatoren?",
+				"answer": "Pretendo wird hautpsächlich für die Hardware der Wii U und des 3DS entwickelt. Zu diesem Zeitpunkt ist CEMU der einzige emulator für diese Konsolen mit NN support. Cemu selbst unterstützt keine eigenen Server, aber es soll trotzdem möglich sein Pretendo mit Cemu zu verwenden.<br>Cemu wird aktuell von Pretendo nicht unterstützt."
+			},
+			{
+				"question": "Wenn ich im Nintendo Network gebannt wurde, bleibe ich in Pretendo auch gebannt?",
+				"answer": "Wir werden keinen Zugriff auf Nintendo's bans haben, weshalt kein Nutzer auf unserem Netzwerk gebannt sein wird. Trotzdem, wir haben Regeln für die Nutzung unserer Dienste und das Nichteinhalten kann in einem Ban resultieren."
+			},
+			{
+				"question": "Wird Pretendo die Wii/Switch unterstützen?",
+				"answer": "Für die Wii gibt es bereits eigene Server von <a href=\"https://wiimmfi.de/\" target=\"_blank\">Wiimmfi</a>. Aktuell möchten wir die Switch auch nicht als Ziel festlegen, da der Online-Service der Switch Kostenpflichtig und  it is both paid and completely different to Nintendo Network."
+			},
+			{
+				"question": "Benötige ich einen Hack zum verbinden?",
+				"answer": "Ja, du musst deine Konsole hacken, allerdings reicht auf der Wii U der Zugriff zum Homebrew Launcher (z.B. über Haxchi, Coldboot Haxchi, oder den Browser-Exploit). Informationen, wie der 3DS verbunden wird, wird später noch veröffentlicht."
+			}
+		]
+	},
+	"credits": {
+		"title": "Das Team",
+		"text": "Triff das Team, das hinter dem Projekt steckt",
+		"people": [
+			{
+				"name": "Jonathan Barrow (RedDucks)",
+				"caption": "Projektinhaber und leitender Entwickler",
+				"picture": "https://github.com/redduckss.png",
+				"github": "https://github.com/RedDuckss"
+			},
```

please change to new username:
```suggestion
			{
				"name": "Jonathan Barrow (jonbarrow)",
				"caption": "Projektinhaber und leitender Entwickler",
				"picture": "https://github.com/jonbarrow.png",
				"github": "https://github.com/jonbarrow"
			},
```